Documentation/project-docs/standard-platform.md

```diff
@@ -26,6 +26,7 @@ The .NET Platform Standard version represents binary portability across platform
 - **.NET Platform Standard** - a specific versioned set of reference assemblies that all .NET Platforms must support as defined in the [CoreFX repo](https://github.com/dotnet/corefx).
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, .NET Framework 4.6, Windows Phone 8.1, MonoTouch, UWP, etc.
```

Comment:
The problem is that .net standard is a different kind of TFM the way it's laid out here. The intent was to call that out in the doc.